module/VuFind/src/VuFind/Db/Service/ShortlinksService.php

```diff
@@ -174,4 +174,21 @@ public function resolve($input, $baseUrl)
         }
         return $baseUrl . current($result)->getPath();
     }
+
+    /**
+     * Generate base62 encoding to migrate old shortlinks
+     *
+     * @return int
```

Comment:
Might be good to edit the comment to explain that the return value is a count of migrated links.